backend/src/bundles/industries/types/industries.controller.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type ApiHandlerResponse } from '~/common/controller/controller.js';
+
+import { type Industry } from './types';
+
+type IIndustriesController = {
```

Comment:
change to interface 
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/92567020/c6818f7e-4679-41d8-bf87-8b234da11b01)